styles/index.css

```diff
@@ -52,3 +116,83 @@
   font-family: var(--heading-sans);
   font-weight: 600;
 }
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+.blogs-header-container h1,
+p {
+  margin: 10px;
+}
+.blogs-header-container h1 {
+  padding: 0;
+}
+.blogs-carousel {
```

Comment:
the ending of arrow is not matching with the content

Reply:
I adjusted the banner-image width:100% to align with the ending of arrow

Reply:
I adjust width: 100% so the arrow and the content aligns well.